synfig-studio/src/gui/widgets/widget_waypoint.cpp

```diff
@@ -269,13 +262,15 @@ Widget_Waypoint::set_waypoint(synfig::Waypoint &x)
 
 	before_options->set_value((Waypoint::Interpolation)waypoint.get_before());
 	after_options->set_value((Waypoint::Interpolation)waypoint.get_after());
-
-	adj_tension->set_value(waypoint.get_tension());
-	adj_continuity->set_value(waypoint.get_continuity());
-	adj_bias->set_value(waypoint.get_bias());
-	adj_temporal_tension->set_value(waypoint.get_temporal_tension());
-
+	
+	if (
```

Comment:
If I understand correctly, this code can now be replaced by:
`Widget_Waypoint::update_tcb_params_visibility()`?

Reply:
Thank you :+1:  Didn't see that. Committing the changes

Reply:
Running studio make... Might take a while before pushing the new changes.